packages/augur-ui/src/reducers/index.ts

```diff
@@ -138,4 +140,5 @@ export interface AppStateInterface {
   versions: Versions;
   accountShareBalances: AccountShareBalances;
   userOpenOrders: OpenOrders;
+  drafts: Drafts;
```

Comment:
I must of missed where `Drafts` interface/type is coming from. I didn't see it in this PR.